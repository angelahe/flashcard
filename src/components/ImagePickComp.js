import React from 'react';
import '../styles/flashcard.css';

class ImagePickComp extends React.Component {
  constructor() {
    super();
    this.state = {
      currentImgUrl: 'https://d30y9cdsu7xlg0.cloudfront.net/png/75112-200.png',
      currentImgId: 75112,
    };
  }

  render() {
    const { currentImgUrl, currentImgId } = this.state;

    return (
      <div>
        <h5>play around with image select</h5>
        <div className="AppPanel">
          <div>{currentImgUrl}</div>
          <div>{currentImgId}</div>
          <div className="cardContainer">
            <img className="cardImg" src={currentImgUrl} alt="nounprojectimg" />
          </div>
        </div>
      </div>
    );
  }
}
export default ImagePickComp;
