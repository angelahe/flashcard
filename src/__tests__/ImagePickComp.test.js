import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import ImagePickComp from '../components/ImagePickComp';

it('renders without crashing', () => {
  shallow(<ImagePickComp />);
});

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<ImagePickComp />, div);
  ReactDOM.unmountComponentAtNode(div);
});

test('ImagePickComp renders correctly', () => {
  const tree = renderer.create(<ImagePickComp />).toJSON();
  expect(tree).toMatchSnapshot();
});
