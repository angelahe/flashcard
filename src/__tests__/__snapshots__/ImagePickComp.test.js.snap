// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImagePickComp renders correctly 1`] = `
<div>
  <h5>
    play around with image select
  </h5>
  <div
    className="AppPanel"
  >
    <div>
      https://d30y9cdsu7xlg0.cloudfront.net/png/75112-200.png
    </div>
    <div>
      75112
    </div>
    <div
      className="cardContainer"
    >
      <img
        alt="nounprojectimg"
        className="cardImg"
        src="https://d30y9cdsu7xlg0.cloudfront.net/png/75112-200.png"
      />
    </div>
  </div>
</div>
`;
